vllm/worker/hpu_model_runner.py

```diff
@@ -756,7 +764,13 @@ def load_model(self) -> None:
             elif not is_fake_hpu():
                 self.model = self.model.to("hpu")
                 htcore.mark_step()
-            modify_decoder_layer(self.model)
+
+            hidden_layer_markstep = int(
```

Comment:
Maybe "hidden_layer_markstep_interval" would be a better name?

Reply:
Updated!